add tasks docs (#84)
resonatehq/docs: docs/operate/cli-reference/commands/tasks.mdx

---
id: tasks
title: CLI tasks command
sidebar_label: Tasks

> description: Mock the behaviour of application node in the Resonate Server.

Reviewer: ```suggestion
description: Interact directly with the Task Framework to mock the behaviour of Application Node.
```

tags:
  - cli
  - tasks
---

> The `tasks` command enables you to interact with the tasks framework by simulating the behavior of an application node.

Reviewer: ```suggestion
The `tasks` command enables you to interact with the Task Framework by simulating the behavior of an Application Node.
```

```shell
resonate tasks <sub-command> <args> [flags]
```

## complete

> Complete a task.

Reviewer: ```suggestion
Complete a Task.
```

```shell
resonate tasks complete <id> [flags]
```

Flags:

- [--counter](/operate/cli-reference/flags#--counter)
- [--process-id](/operate/cli-reference/flags#--process-id)
- [--ttl](/operate/cli-reference/flags#--ttl)

## claim

> Claim one or more tasks associated with the given ID.

Reviewer: ```suggestion
Claim one or more Tasks associated with the given ID.
```

```shell
resonate tasks claim <id> [flags]
```

Flags:

- [--counter](/operate/cli-reference/flags#--counter)

## heartbeat

> Send heartbeat signals for tasks associated with the given process ID.

Reviewer: ```suggestion
Send Heartbeat signals for Tasks associated with the given process ID.
```
